### DISCUSSION TITLE
Deploy without cert manager and webhook?

### DISCUSSION THREAD

**Discussion by nicolasgere** [actions/actions-runner-controller#1348] [Questions]
**Is your feature request related to a problem? Please describe.**
How can we deploy without cert-manager, and all the webhook part. We are not interested in the auto scaling capability.

**Describe the solution you'd like**
An option to disable that part

**Comment by toast-gear** [comment]
What part of the docs don't cover this for you?

**Comment by mumoshu** [comment]
I believe this should better be moved to Discussions.
If you aren't interested in autoscaling, you can just skip creating any HRAs and that's all you'd need.
Re cert-manager, see https://github.com/actions-runner-controller/actions-runner-controller/issues/1029

**Comment by samus-aran** [comment]
We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
